### PR TITLE
chore: add docker build and make targets

### PR DESCRIPTION
**Pull request by olizilla**
- add Dockerfile to build `filecoin/sentinel-visor` docker image
- ~~add make targets to build filecoin-ffi and sentinel-visor~~
- add .dockerignore to reduce context sent to docker build

Updated to use standard make targets from @iand in #18

WIP on https://github.com/filecoin-project/sentinel/issues/141

License: MIT
Signed-off-by: Oli Evans <oli@tableflip.io>

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/.circleci
+/.github
+/extern/filecoin-ffi
+/visor
+/Dockerfile*
+/docker-compose*
+/sentinel-visor
```
Comment (iand) [inline .dockerignore:7]: Line ending?

**Review by iand — APPROVED** [review]
LGTM with a minor nit

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Builder
+FROM golang:1.14.9 as builder
```
Comment (iand) [inline Dockerfile:2]: Why not 1.15?

Comment (olizilla) [reply Dockerfile:2]: This was a guess based on the current go.mod file

https://github.com/filecoin-project/sentinel-visor/blob/47b18855ece919e6c8d337b2a7483764f928090c/go.mod#L3

Comment (iand) [reply Dockerfile:2]: go.mod just defines the minimum version, so it's ok to use a later one

**Pull request merged**